utils/agent_display.py

```
@@ -0,0 +1,166 @@
+            print(f"Error adding message to queue: {e}")
+
+    # clears the messages from a specified layout panel
+    def clear_messages(self, layout_name):
+        """
+        Clears the messages from a specified layout panel.
+        Args:
+            layout_name: The name of the layout panel to clear.
+            Returns:
+            None
+        """
+        if layout_name == "user" or "all":
```

Ruff (0.8.2)

Use `True` instead of `... or True`

Replace with `True`

(SIM222)